apps/condo/domains/notification/templates.test.js

```diff
@@ -236,4 +238,22 @@ describe('Templates', () => {
         expect(preparedMessage.message).not.toMatch(SMS_FORBIDDEN_SYMBOLS_REGEXP)
     })
 
+    it('Password restoration rendered SMS message is not broken and does not contain forbidden symbols (value is normalized)', async () => {
+        const admin = await makeLoggedInAdminClient()
+        const token = faker.datatype.uuid()
+        const [message] = await createTestMessage(admin, {
+            type: RESET_PASSWORD_MESSAGE_TYPE,
```

Comment:
Maybe it's time to write the test which checks all SMS templates?

Reply:
maybe :)